public/styles.css

```diff
@@ -0,0 +1,94 @@
+body { 
```

Comment:
Good use of public/styles to push style choices over all pages. Clean and functional.